<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyScreen</class>
 <widget class="QWidget" name="StudyScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="studySetsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: black;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Return to Study Sets</string>
   </property>
  </widget>
  <widget class="QLabel" name="studySetName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>900</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Insert name</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="questionAnswer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>900</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="showAnswers">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid black;
	border-radius: 10px;
	background-color: white;
}</string>
   </property>
   <property name="text">
    <string>Show Answers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/background.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>studySetsButton</zorder>
  <zorder>studySetName</zorder>
  <zorder>questionAnswer</zorder>
  <zorder>showAnswers</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
